translations/README.it.md

```diff
@@ -4,14 +4,14 @@
 
 ## Indice
 
-- [Informazioni](#about)
-- [Materiale utilizzato](#built-with)
-- [Installazione](#installation)
-- [Demo](#live-demo)
+- [Informazioni](#Informazioni)
```

Comment:
Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `I` to `i`
- [ ] `M` to `m`
- [ ] `I` to `i`
- [ ] `D` to `d`

Reply:
fixed